dev/ScrollViewer/ScrollViewer.cpp

```diff
@@ -162,6 +162,42 @@ void ScrollViewer::InputKind(winrt::InputKind const& value)
     SetValue(s_InputKindProperty, box_value(value));
 }
 
+void ScrollViewer::RegisterAnchorCandidate(winrt::UIElement const& element)
+{
+    SCROLLVIEWER_TRACE_VERBOSE(*this, TRACE_MSG_METH_PTR, METH_NAME, this, element);
+
+    if (auto scroller = m_scroller.get())
+    {
+        const winrt::Controls::IScrollAnchorProvider scrollerAsAnchorProvider = scroller.try_as<winrt::Controls::IScrollAnchorProvider>();
+
+        if (scrollerAsAnchorProvider)
```

Comment:
nit: can simplify to 
if(const auto scrollAnchorProvider  =...)
{
}

Reply:
Done. Thanks.